### ISSUE TITLE
vim-commentary key bindings no longer working

### ISSUE THREAD

**Issue by diminutivesloop** [VSCodeVim/Vim#4410]
**Describe the bug**
Attempting to add or remove comments via the vim-commentary key bindings such as `gcc` no longer works and instead the status bar displays the message “The key combination (G, C) is not a command”.

**Environment (please complete the following information):**

Extension version: 1.12.2
VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T14:57:51.166Z)
OS version: Darwin x64 19.0.0

**Comment by ZaymonFC** [comment]
Work for me.
Version: 1.12.3 
OS: OSX Catalina

**Comment by diminutivesloop** [comment]
Yeah I've also got a windows 10 machine and it still works fine there. But I can't find anything in the config for the installation on my Mac that accounts for the shortcut not working.

**Comment by J-Fields** [comment]
There was some issue introduced in `1.12` that affected only unix users. I'd really appreciate someone investigating it, as I don't currently have access to anything but a windows machine.

**Comment by J-Fields** [comment]
@diminutivesloop can you confirm that this is still an issue in `1.12.4`?

**Comment by diminutivesloop** [comment]
Yes I’m on 1.12.4 and the keybinding still aren’t working.

**Comment by J-Fields** [comment]
You have `vim.vimrc.enable` set to `false`?

**Comment by diminutivesloop** [comment]
Yes that setting is disabled.

**Comment by aomarks** [comment]
Same issue here. vim-commentary feature doesn't work at all, and there seem to be no key binding options for it. VSCode version 1.42.0 for Linux, with VSCodeVim version 1.12.4.

**Comment by diminutivesloop** [comment]
Ok looks like this was caused by some custom keybindings that consisted of ‘g’ and another character chorded together. For example I bound the `editor.action.previewDeclaration` command to `g p`. After converting those keybindings to be defined in `settings.json` under `vim.normalModeKeyBindings` I’m not experiencing this issue anymore.
